lmcache/cache_engine.py

```diff
@@ -257,19 +267,35 @@ def store(
         """
         start_time = time.perf_counter()
         fmt = self.metadata.fmt
-
+        if kv_tensors_mask is None:
+            kv_tensors_mask = torch.ones_like(tokens, dtype=torch.bool)
         assert (len(
             tokens.shape) == 1), f"Invalid shape of tokens: {tokens.shape}"
+        assert (len(
+            kv_tensors_mask.shape) == 1), \
+        f"Invalid shape of mask: {kv_tensors_mask.shape}"
+        assert len(tokens) == len(kv_tensors_mask), \
+            "token length does not match mask length"
+        num_skip_chunk = 0
+        num_skip_tok = 0
+        if kv_tensors_mask is not None:
+            # NOTE(Sixian): Now kv_tensors_mask always a suffix mask.
+            num_skip_tok = (len(kv_tensors_mask) - torch.sum(kv_tensors_mask))
+            num_skip_chunk = num_skip_tok // self.chunk_size
+        assert num_skip_tok == num_skip_chunk * self.chunk_size, \
+            "Only support chunk skip in store."
```

Comment:
What do you mean by "Only support chunk skip in store"? Does it mean the `True`s in `kv_tensors_mask` should align to the chunk boundaries?